src/rust/kafka/src/lib.rs

```diff
@@ -246,7 +247,7 @@ fn consumer(
 }
 
 #[non_exhaustive]
-#[derive(Error, Debug)]
+#[derive(Error, Debug, Clone)]
```

Comment:
These snuck in here as a result of something I thought I had to do, but it ended up being unnecessary. I've left them here because it's probably a good thing to have all these error types be `Clone` anyway.